src/defines.h

```diff
@@ -8,7 +8,6 @@
 #define __LAZY_SOCKETS_VERSION "0.2.4"
```

Comment:
These should be bumped up, version to `0.2.5` and build to the current timestamp